cpu/atmega_common/include/cpu_conf.h

```diff
@@ -49,8 +49,16 @@ extern "C" {
  * to avoid not printing of debug in interrupts
  */
 #ifndef THREAD_STACKSIZE_IDLE
+#if IS_USED(MODULE_XTIMER)
```

Comment:
I get:

```
/data/riotbuild/riotbase/cpu/atmega_common/include/cpu_conf.h:52:12: error: missing binary operator before token "("
 #if IS_USED(MODULE_XTIMER)
```

Reply:
The old AVR toolchain really is annoying :-/ I changed it to `#ifdef` and sqaushed right away.